Source/WebCore/layout/formattingContexts/inline/InlineLineBox.h

```diff
@@ -119,7 +119,7 @@ class LineBox {
     InlineRect m_logicalRect;
     OptionSet<InlineLevelBox::Type> m_boxTypes;
 
-    FontBaseline m_baselineType { AlphabeticBaseline };
+    FontBaseline m_baselineType { FontBaseline::AlphabeticBaseline };
```

Comment:
Can we drop "Baseline" from the value names? it reads pretty redundant now

Reply:
Yup done!